validation/configuration/olomouc.json

```diff
@@ -1,8 +1,7 @@
 {
     "study_region"             : "olomouc",
-    "osm_graphml_path"         : "data/Olomouc/olomouc_cz_2019_10000m_all_osm_20190902.graphml",
-    "osm_buffer_gpkg_path"     : "data/Olomouc/olomouc_cz_2019_1600m_buffer.gpkg",
-    "official_streets_shp_path": "data/Olomouc/olomouc_edges.shp",
-    "official_dests_filepath"  : "data/Olomouc/Olomouc_supermarkets_2018.shp",
-    "dests_column_name"        : "fresh_food_market"
+    "osm_graphml_path"         : "../data/olomouc/olomouc_cz_2019_10000m_all_osm_20190902.graphml",
+    "osm_buffer_gpkg_path"     : "../data/olomouc/olomouc_cz_2019_1600m_buffer.gpkg",
+    "official_streets_shp_path": "../data/olomouc/Olomouc_Street_Network/",
+    "official_dests_filepath"  : "../data/olomouc/olomouc_supermarkets_2018/",
```

Comment:
@nicholas-david the problem is probably this stray trailing comma at the end of this line

Reply:
> Do you know what I should do?

See inline comment on code here.